getflywheel/local-components
# feat (LOC-3986): Build out new combobox component

AC:
- Component is built and implemented to match designs: https://www.figma.com/file/83UXFJyJBu2JFtgJT7mBWX/%F0%9F%92%9A-%F0%9F%9A%80-Local-Finals?node-id=8097%3A21812 --> Done

To test:
- pull and checkout branch
- run yarn build
- run yarn start
- open the browser and select the Storybook tab
- in Storybook, look under Inputs and select Combobox
- select the Connect case
- type, search, select from the dropdown, etc. to test against the designs (see design snapshot below)

<img width="1291" alt="Screen Shot 2022-06-23 at 10 23 23 PM" src="https://user-images.githubusercontent.com/59181296/175404880-0e159649-9199-4f09-a626-d4be06b50826.png">
